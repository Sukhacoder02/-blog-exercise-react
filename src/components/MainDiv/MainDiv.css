#main {
  flex: 1;
  /* background-color: aqua; */
  display: flex;
  align-items: center;
  margin-top: 20px;
  margin-bottom: 20px;
  width: 100vw;
}
