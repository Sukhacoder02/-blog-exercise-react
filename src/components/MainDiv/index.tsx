import * as React from 'react';
import Cards from './../Cards';
import './MainDiv.css';

const MainDiv: React.FunctionComponent = (): JSX.Element => {
  return (
    <div id="main">
      <Cards />
    </div>
  );
};
export default MainDiv;
