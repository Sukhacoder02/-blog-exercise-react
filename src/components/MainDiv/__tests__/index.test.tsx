import * as React from 'react';
import { render } from '@testing-library/react';
import MainDiv from '..';

jest.mock('react-router-dom');
describe('Component MainDiv', () => {
  it('Should render without crashing', () => {
    const { asFragment } = render(<MainDiv />);
    expect(asFragment()).toMatchSnapshot();
  });
});
