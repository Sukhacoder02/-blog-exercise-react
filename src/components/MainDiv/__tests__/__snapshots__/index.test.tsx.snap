// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component MainDiv Should render without crashing 1`] = `
<DocumentFragment>
  <div
    id="main"
  >
    <div
      class="blogDataError"
    >
      <p>
        Loading...
      </p>
    </div>
  </div>
</DocumentFragment>
`;
